### PR TITLE
Bump vault images

### PR DESCRIPTION
**Pull request by ericofusco**
Bump vault image to 1.9.4 and vault k8s to 0.15.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 CHANGES:
 * `global.enabled` now works as documented, that is, setting `global.enabled` to false will disable everything, with individual components able to be turned on individually [GH-703](https://github.com/hashicorp/vault-helm/pull/703)
 * Default value of `-` used for injector and server to indicate that they follow `global.enabled`. [GH-703](https://github.com/hashicorp/vault-helm/pull/703)
-* Vault default image to 1.9.3
+* Vault image default to 1.9.4
```
Comment (tvoran) [inline CHANGELOG.md:6]: Would you mind changing the Vault version to 1.10.0 throughout? That'll be what we want the default at for the next release.

**Comment by tvoran** [comment]
@jseiser No need to change the chart version yet; we'll do that when we do the release.

**Comment by jseiser** [comment]
@tvoran 

Sorry, I assumed the release process was broken or something.

Since the current release contains 

```
https://github.com/hashicorp/vault-helm/blob/v0.19.0/values.yaml#L239
```

But the `main` branch has sat at `1.9.3` since `https://github.com/hashicorp/vault-helm/commit/a7e56d8741a6d1dbeb416215d4677ecf3a8f2a2e`

But there was no matching release that I could find.

**Comment by ericofusco** [comment]
Thank you both. It looks like #719 will update everything. Closing.

**Pull request closed without merging**